src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -98,6 +98,14 @@ trait AttributeComponent {
 
   import driver.api._
 
+  //Magic const we put in listIndex to indicate the dummy row is a list type.
+  val EMPTY_LIST_INDEX_MAGIC = -1
```

Comment:
david an was sad about it, and i care too, so i'll just eat this one and get it done